packages/components/src/draggable/index.js

```diff
@@ -150,7 +154,9 @@ class Draggable extends Component {
 		document.body.classList.add( 'is-dragging-components-draggable' );
 		document.addEventListener( 'dragover', this.onDragOver );
 
-		this.props.setTimeout( onDragStart );
+		this.props.setTimeout(
+			onDragStart.bind( this, event.clientX, event.clientY )
```

Comment:
It might be good to pass the event object rather than the clientX and Y. 

Maybe also the same for `onDragEnd` so that the interface is consistent for each of the callbacks.

Reply:
Unfortunately I couldn't get that to work - something happens to the event object when I pass it, and it ends up with null clientX and clientY props. I don't know enough to about React, javascript or how browsers work to precisely know why. I would love for someone else to fix it because I agree - that API shape ain't the cleanest.

Reply:
@gravityrail It's possibly related to React's event pooling behavior given the event is being passed to a timeout:
https://reactjs.org/docs/events.html#event-pooling

I'll do some debugging.

Reply:
Yep, it was related to the event pooling.

I pushed a commit that makes the change. If you want to `git fetch` from this repo and then `git cherry-pick db82900` will bring it onto your branch.

Reply:
Done! Thanks for the patch 👍